### PR TITLE
editoast: add some doc about batch updating dependencies

### PR DESCRIPTION
**Pull request by Castavo**
To be merged after osrd-project/osrd-website#142

### CODE REVIEW

File: editoast/Cargo.toml [hunk 1]
```diff
@@ -15,9 +15,18 @@ serde_derive = "1.0.188"
...
+# Batch version bumping process:
+# 1. If you're using VSCode, you can install the `crates` extension and run the "update all dependencies" command. 
+#   Doing so will update all dependencies to their latest version, and overwrite voluntarily loose version constraints.
+#   Make sure that the new version chosen is stable, and that loose constraints are not overwritten in your commit.
+# 1 alt. If you're not, you can go check the versions used by dependabot in [its PRs](https://github.com/osrd-project/osrd/pulls?q=is%3Aopen+label%3Aarea%3Aeditoast+label%3Adependencies)
+#   and update the versions manually.
+# 2. Run `cargo update` to update the Cargo.lock file (even sub-dependencies)
+# 3. Check that all dependabot editoast PRs are included in your update
+# 4. Adapt the code to the new versions, if needed
+# 5. Create a PR with your changes, and link all dependabot PRs in the description
```
Comment (flomonster) [inline editoast/Cargo.toml]: Actually, I find it odd to put this here.
We could move this to the README or osrd.fr documentation (in a "How to" section)?

Comment (Castavo) [reply editoast/Cargo.toml]: OK, what do you think of putting it in osrd.fr and mention the doc in this file in a one line comment ?

**Review by flomonster — APPROVED** [review]
Nice. Should wait for this [PR](https://github.com/osrd-project/osrd-website/pull/142).

**Comment by hamz2a** [comment]
LGTM

**Review by Tguisnet — APPROVED** [review]

**Pull request merged**